cli/src/utils.ts

```diff
@@ -73,3 +76,48 @@ export function checkAPIKey() {
     );
   }
 }
+
+export const checkForUpdates = async () => {
+  try {
+    if (config.disableUpdateCheck === 'true') {
+      return;
+    }
+
+    const currentTime = Date.now();
+
+    const data = existsSync(configFile) ? yaml.load(readFileSync(configFile, 'utf8')) ?? {} : {};
+    const configFileData = JSON.parse(JSON.stringify(data));
```

Comment:
We should deal with the config in a type-safe way.

Reply:
Done